docs/agents.md

```diff
@@ -19,7 +18,9 @@ def get_weather(city: str) -> str:
     return f"The weather in {city} is sunny"
 
 agent = Agent(
+    # The 'name' is the agent's identity, used for context and tool calling.
```

Comment:
What I meant was mentioning "required" or not for these two parameters rather than adding these description